packages/ember/tests/routing/query_params_test/shared_state_test.js

```diff
@@ -2,10 +2,10 @@ import {
   Controller,
   Service
 } from 'ember-runtime';
-import Ember from 'ember';
 import { run } from 'ember-metal';
 import { jQuery } from 'ember-views';
 import { QueryParamTestCase, moduleFor } from 'internal-test-helpers';
+import { inject } from '@ember/service';
```

Comment:
Can you change this to `import { inject as service } from '@ember/service';`

Reply:
@locks Done 😄